From ce53d1aa526c083bbdb7e1cce76f7d8bdc06fb0a Mon Sep 17 00:00:00 2001
From: Sergey V. Udaltsov <svu@gnome.org>
Date: Wed, 25 Feb 2009 21:30:55 +0000
Subject: missing minus/underscore added to ru(phonetic), b.fd.o#20093

---
diff --git a/symbols/ru b/symbols/ru
index dda7281..e12129b 100644
--- a/symbols/ru
+++ b/symbols/ru
@@ -146,6 +146,7 @@ xkb_symbols "phonetic" {
     key	<AE08> {	[		8,        asterisk	]	};
     key	<AE09> {	[		9,	 parenleft	]	};
     key	<AE10> {	[		0,	parenright	]	};
+    key <AE11> {	[	    minus,	underscore	]	};
 
     key	<AB09> {	[	   period,	   greater	]	};
     key	<AB10> {	[	    slash,	  question	]	};
--
cgit v0.8.2
