From 59a3190b9541b77eeba23609b74a767e996bce68 Mon Sep 17 00:00:00 2001
From: Sergey V. Udaltsov <svu@linnie-the-pooh.(none)>
Date: Mon, 16 Mar 2009 09:18:32 +0000
Subject: fixed Kyrgyz layouts, b.fd.o#17505

---
diff --git a/ChangeLog b/ChangeLog
index e3062be..1f5cc92 100644
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,3 +1,8 @@
+2009-03-16 Sergey Udaltsov
+
+	* rules/base.lists.part, rules/base.xml.in, symbols/kg: fixed Kyrgyz
+	variants, http://bugs.freedesktop.org/show_bug.cgi?id=17505
+
 2009-03-07 Sergey Udaltsov
 
 	* rules/base.m_s.part, rules/base.xml.in: added trust_slimline model,
diff --git a/rules/base.lists.part b/rules/base.lists.part
index b708e46..7e75dfc 100644
--- a/rules/base.lists.part
+++ b/rules/base.lists.part
@@ -74,7 +74,6 @@
               gb gr \
               hu \
               ie ir is it \
-              kg \
               latam \
               lk lt \
               mn mt \
diff --git a/rules/base.xml.in b/rules/base.xml.in
index a7616c5..3f11c9a 100644
--- a/rules/base.xml.in
+++ b/rules/base.xml.in
@@ -2826,7 +2826,14 @@
         <_description>Kyrgyzstan</_description>
         <languageList><iso639Id>kir</iso639Id></languageList>
       </configItem>
-      <variantList/>
+      <variantList>
+       <variant>
+          <configItem>
+            <name>phonetic</name>
+            <_description>Phonetic</_description>
+          </configItem>
+        </variant>
+      </variantList>
     </layout>
     <layout>
       <configItem>
diff --git a/symbols/kg b/symbols/kg
index 24f5a15..993949a 100644
--- a/symbols/kg
+++ b/symbols/kg
@@ -1,28 +1,135 @@
 // $XKeyboardConfig$
 
-// based on
-// russian standard keyboard
-// Yury Fedorov <yury-fedorov@prime-task.com>
-// Last Changes 2004/10/25 by Yury Fedorov <yury-fedorov@prime-task.com>
+// Kirghiz (aka Kyrgyz) standard keyboard
 //
-// $XFree86: xc/programs/xkbcomp/symbols/pc/kg,v 0.0 Exp $
+// Standard	 -	Phonetic
+// ______________- ________________
+// ё1234567890өң     ё1234567890яю
+// Ё!"#*:,.;()ӨҢ     Ё!"№;%:?-()ЯЮ
+//
+// йцукенгшщзхъү     өүертыуиопчшъ
+// фывапролджэ       асдфгңйклэь
+// ячсмитьбю/(?)     зхжвбнмцщ.(,)
 
-partial default alphanumeric_keys 
-xkb_symbols "basic" {
+// Authors:
+// Timur Jamakeev <ztimur [at] gmail.com>
+// Ilyas Bakirov <just_ilyas [at] yahoo.com>
+// Yury Fedorov <yury-fedorov [at] prime-task.com>
+// Emil Asanov <a7 [at] on.kg>
+// Ulan Melisbek <ulan.melisbek [at] gmail.com>
 
-    include "ru(winkeys)"
 
-    name[Group1]= "Kyrgyzstan";
+// Kyrgyz standard keyboard
+partial default alphanumeric_keys
+xkb_symbols "basic" {
 
-    key.type[group1]="FOUR_LEVEL_ALPHABETIC";
+    name[Group1]= "Kyrgyzstan";
 
-    key	<AC07> { [ 	Cyrillic_o,		Cyrillic_O,		
-			Cyrillic_o_bar,  	Cyrillic_O_bar	] };
-    key	<AD03> { [ 	Cyrillic_u,		Cyrillic_U,
-			Cyrillic_u_straight,	Cyrillic_U_straight	] };
-    key	<AD06> { [ 	Cyrillic_en, 		Cyrillic_EN,	
-			Cyrillic_en_descender,	Cyrillic_EN_descender	] };		
+    key	<AE01> {	[		1,	    exclam 	]	};
+    key	<AE02> {	[		2,        quotedbl	]	};
+    key	<AE03> {	[		3,      numerosign	]	};
+    key	<AE04> {	[		4,        asterisk	]	};
+    key	<AE05> {	[		5,	     colon	]	};
+    key	<AE06> {	[		6,	     comma	]	};
+    key	<AE07> {	[		7,	    period	]	};
+    key	<AE08> {	[		8,	 semicolon	]	};
+    key	<AE09> {	[		9,	 parenleft	]	};
+    key	<AE10> {	[		0,	parenright	]	};
+    key	<AE11> {	[    Cyrillic_o_bar,  		Cyrillic_O_bar		]	};
+    key	<AE12> {	[    Cyrillic_en_descender,   	Cyrillic_EN_descender	]	};
+    key	<BKSL> {	[    Cyrillic_u_straight,  	Cyrillic_U_straight	]	};
+    key	<AB10> {	[	    slash,	  question	]	};
+    key <LSGT> {        [           slash,             bar      ]       };
 
-    include "level3(ralt_switch)"
+    key	<TLDE> {	[     Cyrillic_io,     Cyrillic_IO	]	};
+    key	<AD01> {	[ Cyrillic_shorti, Cyrillic_SHORTI	]	};
+    key	<AD02> {	[    Cyrillic_tse,    Cyrillic_TSE	]	};
+    key	<AD03> {	[      Cyrillic_u,	Cyrillic_U	]	};
+    key	<AD04> {	[     Cyrillic_ka,     Cyrillic_KA	]	};
+    key	<AD05> {	[     Cyrillic_ie,     Cyrillic_IE	]	};
+    key	<AD06> {	[     Cyrillic_en,     Cyrillic_EN	]	};
+    key	<AD07> {	[    Cyrillic_ghe,    Cyrillic_GHE	]	};
+    key	<AD08> {	[    Cyrillic_sha,    Cyrillic_SHA	]	};
+    key	<AD09> {	[  Cyrillic_shcha,  Cyrillic_SHCHA	]	};
+    key	<AD10> {	[     Cyrillic_ze,     Cyrillic_ZE	]	};
+    key	<AD11> {	[     Cyrillic_ha,     Cyrillic_HA	]	};
+    key	<AD12> {	[	Cyrillic_hardsign,	Cyrillic_HARDSIGN	]	};
+    key	<AC01> {	[     Cyrillic_ef,     Cyrillic_EF	]	};
+    key	<AC02> {	[   Cyrillic_yeru,   Cyrillic_YERU	]	};
+    key	<AC03> {	[     Cyrillic_ve,     Cyrillic_VE	]	};
+    key	<AC04> {	[      Cyrillic_a,	Cyrillic_A	]	};
+    key	<AC05> {	[     Cyrillic_pe,     Cyrillic_PE	]	};
+    key	<AC06> {	[     Cyrillic_er,     Cyrillic_ER	]	};
+    key	<AC07> {	[      Cyrillic_o,	Cyrillic_O	]	};
+    key	<AC08> {	[     Cyrillic_el,     Cyrillic_EL	]	};
+    key	<AC09> {	[     Cyrillic_de,     Cyrillic_DE	]	};
+    key	<AC10> {	[    Cyrillic_zhe,    Cyrillic_ZHE	]	};
+    key	<AC11> {	[      Cyrillic_e,	Cyrillic_E	]	};
+    key	<AB01> {	[     Cyrillic_ya,     Cyrillic_YA	]	};
+    key	<AB02> {	[    Cyrillic_che,    Cyrillic_CHE	]	};
+    key	<AB03> {	[     Cyrillic_es,     Cyrillic_ES	]	};
+    key	<AB04> {	[     Cyrillic_em,     Cyrillic_EM	]	};
+    key	<AB05> {	[      Cyrillic_i,	Cyrillic_I	]	};
+    key	<AB06> {	[     Cyrillic_te,     Cyrillic_TE	]	};
+    key	<AB07> {	[ 	Cyrillic_softsign,	Cyrillic_SOFTSIGN	]	};
+    key	<AB08> {	[     Cyrillic_be,     Cyrillic_BE	]	};
+    key	<AB09> {	[     Cyrillic_yu,     Cyrillic_YU	]	};
 };
 
+// Kyrgyz phonetic layout
+partial alphanumeric_keys
+xkb_symbols "phonetic" {
+
+    name[Group1]="Kyrgyzstan - Phonetic";
+
+    key	<AE01> {	[		1,	    	exclam 			]	};
+    key	<AE02> {	[		2,        	quotedbl		]	};
+    key	<AE03> {	[		3,      	numerosign		]	};
+    key	<AE04> {	[		4,     	 	semicolon		]	};
+    key	<AE05> {	[		5,    	   	percent			]	};
+    key	<AE06> {	[		6, 	     	colon			]	};
+    key	<AE07> {	[		7,	  	question		]	};
+    key	<AE08> {	[		8,           	minus			]	};
+    key	<AE09> {	[		9,	 	parenleft		]	};
+    key	<AE10> {	[		0,		parenright		]	};
+    key	<AE11> {	[    	Cyrillic_ya,     	Cyrillic_YA		]	};
+    key	<AE12> {	[    	Cyrillic_yu,     	Cyrillic_YU		]	};
+
+    key	<AD11> {	[    	Cyrillic_che,		Cyrillic_CHE		]	};
+    key	<AD12> {	[    	Cyrillic_sha,    	Cyrillic_SHA		]	};
+    key	<BKSL> {	[	Cyrillic_hardsign, 	Cyrillic_HARDSIGN	]	};
+    key	<AB09> {	[    	Cyrillic_shcha,    	Cyrillic_SHCHA		]	};
+    key	<AB10> {	[      	period, 		comma			]	};
+    key	<AB08> {	[	Cyrillic_tse, 		Cyrillic_TSE		]	};
+    key	<AC10> {	[	Cyrillic_e,        	Cyrillic_E		]	};
+    key	<AC11> {	[	Cyrillic_softsign, 	Cyrillic_SOFTSIGN	]	};
+    key	<TLDE> {	[     	Cyrillic_io,     	Cyrillic_IO		]	};
+
+    key	<LatQ> {	[     	Cyrillic_o_bar,  	Cyrillic_O_bar		]	};
+    key	<LatZ> {	[     	Cyrillic_ze,     	Cyrillic_ZE		]	};
+    key	<LatS> {	[     	Cyrillic_es,     	Cyrillic_ES		]	};
+    key	<LatA> {	[     	Cyrillic_a,		Cyrillic_A		]	};
+    key	<LatW> {	[     	Cyrillic_u_straight,  	Cyrillic_U_straight	]	};
+    key	<LatC> {	[     	Cyrillic_zhe,    	Cyrillic_ZHE		]	};
+    key	<LatX> {	[	Cyrillic_ha,     	Cyrillic_HA 		]	};
+    key	<LatD> {	[     	Cyrillic_de,		Cyrillic_DE		]	};
+    key	<LatE> {	[     	Cyrillic_ie,     	Cyrillic_IE		]	};
+    key	<LatV> {	[     	Cyrillic_ve,     	Cyrillic_VE		]	};
+    key	<LatF> {	[     	Cyrillic_ef,     	Cyrillic_EF		]	};
+    key	<LatT> {	[     	Cyrillic_te,     	Cyrillic_TE		]	};
+    key	<LatR> {	[     	Cyrillic_er,     	Cyrillic_ER		]	};
+    key	<LatN> {	[     	Cyrillic_en,     	Cyrillic_EN		]	};
+    key	<LatB> {	[     	Cyrillic_be,     	Cyrillic_BE		]	};
+    key	<LatH> {	[     	Cyrillic_en_descender,  Cyrillic_EN_descender	]	};
+    key	<LatG> {	[    	Cyrillic_ghe,    	Cyrillic_GHE		]	};
+    key	<LatY> {	[   	Cyrillic_yeru,   	Cyrillic_YERU		]	};
+    key	<LatM> {	[     	Cyrillic_em,     	Cyrillic_EM		]	};
+    key	<LatJ> {	[ 	Cyrillic_shorti, 	Cyrillic_SHORTI		]	};
+    key	<LatU> {	[      	Cyrillic_u,		Cyrillic_U		]	};
+    key	<LatK> {	[     	Cyrillic_ka,     	Cyrillic_KA		]	};
+    key	<LatI> {	[      	Cyrillic_i,		Cyrillic_I		]	};
+    key	<LatO> {	[      	Cyrillic_o,		Cyrillic_O		]	};
+    key	<LatL> {	[     	Cyrillic_el,     	Cyrillic_EL		]	};
+    key	<LatP> {	[     	Cyrillic_pe,     	Cyrillic_PE		]	};
+
+};
--
cgit v0.8.2
