Index: xkeyboard-config-0.9/rules/base.xml.in
===================================================================
--- xkeyboard-config-0.9.orig/rules/base.xml.in	2008-01-31 08:38:13.000000000 -0400
+++ xkeyboard-config-0.9/rules/base.xml.in	2008-01-31 08:38:36.000000000 -0400
@@ -1235,6 +1235,12 @@
             <_description>Inuktitut</_description>
           </configItem>
         </variant>
+        <variant>
+          <configItem>
+            <name>shs</name>
+            <_description>Secwepemctsin</_description>
+          </configItem>
+        </variant>
       </variantList>
     </layout>
     <layout>
Index: xkeyboard-config-0.9/symbols/ca
===================================================================
--- xkeyboard-config-0.9.orig/symbols/ca	2008-01-31 08:38:46.000000000 -0400
+++ xkeyboard-config-0.9/symbols/ca	2008-01-31 08:39:26.000000000 -0400
@@ -453,3 +453,27 @@
     include "level5(rctrl_switch)"
 };
 
+// Secwepemctsin keyboard for X11
+// written by Neskie Manuel <neskiem@gmail.com>
+
+partial alphanumeric_keys
+xkb_symbols "shs" {
+
+    include "us"
+
+    name[Group1]= "Canada - Secwepemctsin";
+
+    key <AD01> {	[   q,	Q, UF7EB, UF7EA ]	};
+    key <AD02> {	[   w,	W, UF83B, UF83A ]	};
+    key <AD04> {	[   r,	R, UF7FB, UF7FA ]	};
+    key <AD05> {	[   t,	T, UF81B, UF81A ]	};
+    key <AD06> {	[   y,	Y, UF6AB, UF6AA ]	};
+    key <AD10> {	[   p,	P, UF7DB, UF7DA ]	};
+    key <AC02> {	[   s,	S, UF80B, UF80A	]	};
+    key <AC05> {	[   g,	G, UF76B, UF76A	]	};
+    key <AC08> {	[   k,	K, UF79B, UF79A ]	};
+    key <AC09> {	[   l,	L, UF7AB, UF7AA ]	};
+    key <AB07> {	[   m,	M, UF7BB, UF7BA ]	};
+
+    include "level3(ralt_switch)"
+};
