Description: Addition of an Elfdalian keyboard layout
Origin: https://cgit.freedesktop.org/xkeyboard-config/commit/?id=8712fb1210a0bab8f72a72bda20c17d52b172fd3
Bug: https://launchpad.net/bugs/1655782
Author: Gunnar Hjalmarsson <gunnarhj@ubuntu.com>

diff -ru a/rules/base.extras.xml.in b/rules/base.extras.xml.in
--- a/rules/base.extras.xml.in	2016-01-14 02:02:13.000000000 +0100
+++ b/rules/base.extras.xml.in	2016-10-10 02:13:24.455895148 +0200
@@ -751,6 +751,15 @@
             <_description>Swedish (Sun Type 6/7)</_description>
           </configItem>
         </variant>
+        <variant>
+          <configItem>
+            <name>ovd</name>
+            <_description>Elfdalian (Swedish, with combining ogonek)</_description>
+            <languageList>
+              <iso639Id>ovd</iso639Id>
+            </languageList>
+          </configItem>
+        </variant>
       </variantList>
     </layout>
     <layout>
diff -ru a/rules/evdev.extras.xml.in b/rules/evdev.extras.xml.in
--- a/rules/evdev.extras.xml.in	2016-01-14 02:02:16.000000000 +0100
+++ b/rules/evdev.extras.xml.in	2016-10-10 02:13:24.455895148 +0200
@@ -751,6 +751,15 @@
             <_description>Swedish (Sun Type 6/7)</_description>
           </configItem>
         </variant>
+        <variant>
+          <configItem>
+            <name>ovd</name>
+            <_description>Elfdalian (Swedish, with combining ogonek)</_description>
+            <languageList>
+              <iso639Id>ovd</iso639Id>
+            </languageList>
+          </configItem>
+        </variant>
       </variantList>
     </layout>
     <layout>
diff -ru a/symbols/se b/symbols/se
--- a/symbols/se	2014-02-10 03:07:48.000000000 +0100
+++ b/symbols/se	2016-10-10 02:13:24.455895148 +0200
@@ -359,4 +359,18 @@
 
 };
 
+partial alphanumeric_keys
+xkb_symbols "ovd" {
 
+    // Adds combining ogonek (U0328) to the basic Swedish layout to facilitate
+    // typing of Ą ą Ę ę Į į Ų ų Y̨ y̨ Ą̊ ą̊.
+    // The character replaces <dead_diaeresis> which is considered dispensable
+    // since ä and ö have their own keys. <dead_diaeresis> is still available
+    // via <AltGr> + Å.
+
+    include "se(basic)"
+
+    name[Group1]="Elfdalian (Swedish, with combining ogonek)";
+
+    key <AD12> { [ U0328,  dead_circumflex,  dead_tilde,  dead_caron ] };
+};
