diff -Naur xkeyboard-config.orig/rules/base.xml.in xkeyboard-config/rules/base.xml.in
--- xkeyboard-config.orig/rules/base.xml.in	2007-08-11 12:10:28.335198808 +0300
+++ xkeyboard-config/rules/base.xml.in	2007-08-11 12:51:22.618434296 +0300
@@ -2528,12 +2528,24 @@
       <variantList>
         <variant>
           <configItem>
+            <name>comma</name>
+            <_description>Commabelow</_description>
+          </configItem>
+        </variant>
+        <variant>
+          <configItem>
             <name>std</name>
             <_description>Standard</_description>
           </configItem>
         </variant>
         <variant>
           <configItem>
+            <name>academic</name>
+            <_description>Standard (Commabelow)</_description>
+          </configItem>
+        </variant>
+        <variant>
+          <configItem>
             <name>winkeys</name>
             <_description>Winkeys</_description>
           </configItem>
